smoggnd.pro

```diff
@@ -2,8 +2,8 @@ TEMPLATE = app
 
 QT += qml quick widgets multimedia network serialport charts quickcontrols2
 
-lessThan(QT_MAJOR_VERSION, 5) | lessThan(QT_MINOR_VERSION, 12) {
-    error("Qt 5.12 is the minimum required version.")
+lessThan(QT_MAJOR_VERSION, 5) | lessThan(QT_MINOR_VERSION, 12) | lessThan(QT_PATCH_VERSION, 6) {
```

Comment:
Won't this also reject versions like 5.13.5 or 5.14.0 or anything with a less than 6 patch number?

Reply:
You are right, this slipped my mind. This will be fixed with commit fd85a1b91c9fb8faa4d132620f08e081d230b0d1